README.md

```diff
@@ -10,8 +10,6 @@ DATASIM provides DoD distributed learning stakeholders and the broader xAPI comm
 
 DATASIM is funded by the Advanced Distributed Learning Initiative at US DoD.
 
-This documentation and repository refer to the simulation engine of DATASIM, which will run as a standalone CLI, and may also be deployed as a REST API. For the User Interface (which interacts with the API) please see https://github.com/yetanalytics/datasim-ui.
```

Comment:
The first sentence is still true, so we can just keep that and only remove the second sentence, instead of removing the entire line.

Reply:
Reason for deleting the whole line discussed in Slack.